RecoCTPPS/PixelLocal/test/run_only_CTPPS_cfg_CLU_DB_real_mem.py

```diff
@@ -0,0 +1,93 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.StandardSequences.Eras import eras
+
+process = cms.Process('CTPPS2',eras.Run2_25ns)
+
+# import of standard configurations
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('FWCore.MessageService.MessageLogger_cfi')
+process.load('Configuration.EventContent.EventContent_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+
+process.maxEvents = cms.untracked.PSet(
+        input = cms.untracked.int32(10000)
+        )
+
+process.MessageLogger = cms.Service("MessageLogger",
+    destinations = cms.untracked.vstring('cclu_info'),
+    cclu_info = cms.untracked.PSet( threshold = cms.untracked.string('INFO'))
+)
+process.source = cms.Source("EmptyIOVSource",
+    timetype = cms.string('runnumber'),
+    firstValue = cms.uint64(1),
+    lastValue = cms.uint64(1),
+    interval = cms.uint64(1)
+)
+
+# Track memory leaks
+process.SimpleMemoryCheck = cms.Service("SimpleMemoryCheck",ignoreTotal = cms.untracked.int32(1) )
+
+
+process.source = cms.Source("PoolSource",
+    fileNames = cms.untracked.vstring(
+'root://cms-xrd-global.cern.ch//store/data/Run2017A/ZeroBias2/AOD/PromptReco-v1/000/294/736/00000/44589413-F73F-E711-9E8D-02163E014337.root'
+
+),
+duplicateCheckMode = cms.untracked.string("checkEachFile")
+)
+
+
+from CondCore.CondDB.CondDB_cfi import *
+process.PoolDBESSource = cms.ESSource("PoolDBESSource",
+        CondDB.clone(
+connect = cms.string('frontier://FrontierPrep/CMS_CONDITIONS')),
+    toGet = cms.VPSet(cms.PSet(
+        record = cms.string('CTPPSPixelGainCalibrationsRcd'),
+        tag = cms.string("CTPPSPixelGainCalibNew_v4_test")
+    ),
+      cms.PSet(
+        record = cms.string('CTPPSPixelDAQMappingRcd'),
+        tag = cms.string("CTPPSPixelDAQMapping_v1_test"),
+        label = cms.untracked.string("RPix")
+      ),
+      cms.PSet(
+        record = cms.string('CTPPSPixelAnalysisMaskRcd'),
+        tag = cms.string("CTPPSPixelAnalysisMask_v1_test"),
+        label = cms.untracked.string("RPix")
+      )
+
```

Comment:
picking specific tags
modifying existing global tags 
is a practice firmly discourage (traceability of what conditions are served is easily lost).
We gather this file is only for test, still, now that you can use GT, please try to adhere to the policy.

Also here applies the comment about "RPix"